src/screens/FearConditioningTrialScreen.tsx

```diff
@@ -51,23 +52,42 @@ export const FearConditioningTrialScreen: React.FunctionComponent<FearConditioni
     anchorLabelRight,
   }) => {
     const Alert = useAlert()
+    const focusState = useSelector((state) => state.appState)
     const [showTrial, setShowTrial] = useState<boolean>(false)
     const [showScale, setShowScale] = useState(false)
 
     // Keep reference of timers
     const startTimeRef = useRef<number>()
     const reactionTimeRef = useRef<number>()
-    const endTimerRef = useRef<any>()
-    const soundTimerRef = useRef<any>()
-    const scaleTimerRef = useRef<any>()
+    const endTimerRef = useRef<PauseableTimer>()
+    const soundTimerRef = useRef<PauseableTimer>()
+    const scaleTimerRef = useRef<PauseableTimer>()
+    const mountedTimerRef = useRef<PauseableTimer>()
```

Comment:
Suggestion: This is perfectly fine.

But I think it would be nicer if it was a React hook.

Reply:
I agree, I'll note down and come back to as this PR has bug fixes we need :+1: